assets/js/product.js

```diff
@@ -16,82 +16,12 @@ storefrontApp.controller('productController', ['$rootScope', '$scope', '$window'
         $scope.configurationQty = 1;
         $scope.validateQtyInput = validationHelper.positiveInt;
 
-        // TODO: Replace mock with real function
-        $scope.addProductsToCartMock = function () {

```

Comment:
Dead code too. I was need to change "Added to product cart" dialog here too, but found it isn't used anywhere